app/api/routers/breeze_buddy/langfuse/__init__.py

```
@@ -0,0 +1,174 @@
+        logger.info("=== RAW LANGFUSE RESPONSE ===")
+        logger.info(f"{raw_response}")
+        logger.info("=== END RAW RESPONSE ===")
+
+        return {
+            "success": True,
+            "from_time": from_time.isoformat(),
+            "to_time": to_time.isoformat(),
+            "raw_response": raw_response,
+        }
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)